Rego/DefenderConfig.rego

```diff
@@ -148,45 +125,47 @@ tests[{
 #--
 
 # TODO check exclusions
```

Comment:
Do we have an issue on this? It is repeated several times.

Reply:
I believe we do, I am going to have to dig through the closed issues/PRs to confirm tho because I did not write the checks, just cleaning.

Reply:
Yea, out of scope on address.  I only glanced but did not see exclusion checking code.  If closed then TODO should be removed - it TODO is valid then an open issue should exist and be reference in TODO (maybe - could read template checklist a couple way - a bit ambiguous).